debian/common_context/entrypoint.sh

```diff
@@ -9,27 +9,44 @@
 #   Felix Schindler (2017)
 #   Rene Milk       (2017)
 
-USERNAME_=${LOCAL_USER:-user}
-UID_=${LOCAL_UID:-1000}
+USERNAME_=${LOCAL_USER:-root}
+UID_=${LOCAL_UID:-0}
 GID_=${LOCAL_GID:-$UID_}
 
-groupadd -g $GID_ $USERNAME_ &> /dev/null
-if [ -e /home/$USERNAME_ ] ; then
-  useradd -d /home/$USERNAME_ -g $GID_ -s /bin/bash -u $UID_ $USERNAME_
-else
-  useradd -md /home/$USERNAME_ -g $GID_ -s /bin/bash -u $UID_ $USERNAME_
-fi
+export LANG=en_US.UTF-8
+echo "127.0.0.1 ${HOSTNAME}" >> /etc/hosts
```

Comment:
Why is this necessary?

Reply:
Actually, I do not recall. It was present in the entrypoint before so I just kept it.

Reply:
Ah, further down in the diff, it moved up. Maybe this is useful if one sets the hostname at `docker run` time. DIdn't think of that earlier.

Reply:
Yup, our docker_run.sh sets the hostname, so it fits well I think